source/_docs/automation/trigger.markdown

```diff
@@ -99,7 +99,7 @@ automation:
 
 <div class='note'>
 Listing above and below together means the numeric_state has to be between the two values.
-In the example above, the trigger would fire if a numeric_state goes to 17.1-24.9 (from strict above 17 and strict below 25).
+In the example above, the trigger would fire if a numeric_state goes to 17.1-24.9 (from 17 and below or 25 and above).
```

Comment:
`and above`?

You are saying it triggers on 25.1?

Reply:
I don't know how to say it better: it fires when goes from (-inf, 17] to (17,25) or from [25, +inf) to (17,25). That's why I wrote "25 and above".

Reply:
Aah... you trying to tell it only fires once it enters the defined threshold!

Now I get it. Maybe we should say that?

Reply:
Yes, but my understanding is that author of this note wanted to be precise when it comes to strict inequalities.

Reply:
yes, but from your change, I would not make it clear either (at least not for me)

Reply:
I agree, but nothing better comes to my mind 😞

Reply:
I keep my fingers crossed...

Reply:
```suggestion
In the example above, the trigger would fire a single time if a numeric_state goes into to 17.1-24.9 rang (from 17 and below or 25 and above). It will only fire again, once it has left the defined range and enters it again.
```